myWidget/Views/myWidgetEntryView.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  myWidgetEntryView.swift
+//  포스밀
+//
+//  Created by Kim Insub on 2022/07/23.
+//
+
+import SwiftUI
+import WidgetKit
+
+struct myWidgetEntryView : View {
```

Comment:
Swift style guide에서 콜론 `:` 은 뒤에만 스페이스 하나가 권장되어요
```suggestion
struct myWidgetEntryView: View {
```

Reply:
확인했습니다~